src/backend/dx12/src/device.rs

```diff
@@ -2193,7 +2274,16 @@ impl d::Device<B> for Device {
                         dst_samplers.push(bind_info.sampler_range.as_ref().unwrap().at(offset));
                         num_samplers.push(1);
                     }
-                    pso::Descriptor::TexelBuffer(_) => unimplemented!()
+                    pso::Descriptor::UniformTexelBuffer(buffer_view) => {
+                        src_views.push(buffer_view.handle_srv);
```

Comment:
given that the SRV here and UAV below can be NULL, can we at least `error!()` on it?